src/Form/OchaAiChatChatForm.php

```diff
@@ -263,9 +231,10 @@ public function buildForm(array $form, FormStateInterface $form_state, ?bool $po
 
     $form['question'] = [
       '#type' => 'textarea',
-      '#title' => $this->t('Question'),
+      '#title' => $this->t('Enter your question'),
+      '#title_display' => 'invisible',
       '#default_value' => $form_state->getValue('question') ?? NULL,
-      '#description' => $this->t('Ex: How many people are in need of humanitarian assistance in <em>location</em> due to the <em>event</em> that started on <em>date</em>?'),
+      '#placeholder' => $this->t('Ex: How many people are in need of nutritional assistance?'),
```

Comment:
```suggestion
      '#placeholder' => $this->t('Ex: How many people are in need of food assistance?'),
```

Nitpicking but "Nutritional assistance" doesn't seem like a very common term ([~500 documents](https://reliefweb.int/updates?search=%22nutritional+assistance%22)). "Food assistance" ([~31,000 documents](https://reliefweb.int/updates?search=%22food+assistance%22)) may be better.

Reply:
Good idea. I only picked that example based on the particular article I'd been viewing.